lmdeploy/pytorch/check_env/__main__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
personally, not suggest to add `__main__.py`. Could we make it into cli as  `lmdeploy check_env --backend pytorch`?

Reply:
This is a debug-only api for developer, not the final user.